docs/proposals/refactor-keadm-on-the-cloud.md

```diff
@@ -49,7 +49,7 @@ spec:
           - matchExpressions:
           - key: node-role.kubernetes.io/edge
               operator: DoesNotExist
-    tolerations: {}
+    tolerations: []
     nodeSelector: {}
```

Comment:
@zhu733756   one question, here nodeSelector is an object, how to specify its value in the command line during keadm installation. :)

Reply:
Can you try this? 

![image](https://user-images.githubusercontent.com/38685176/187333805-4c69b8d7-8dd2-4130-b816-e891837bb8ff.png)

Reply:
yes, it works, thanks :)